Fixed #34822 -- Added support for serializing functions decorated with functools.lru_cache in migrations. (#17256)
django/django: docs/releases/5.0.txt

Migrations
~~~~~~~~~~

> * Migrations now support serialization of functions decorated with

Reviewer: On a second thought, I think it would be a good idea to remove "Migrations now" since the section title is specific about Migrations. So:

```suggestion
* Support for serialization of functions decorated with... was added.
```

Author: I copied from the 4.2 release notes where `EnumFlag` was added. Based on that would you still prefer the change?

Author: I made the suggested change anyway.

> * Support for serialization of functions decorated with :func:`functools.cache`

Reviewer 2: ```
`* Migrations now support serialization of functions ...
```

Author: That's what I had until https://github.com/django/django/pull/17256#discussion_r1324732501... I'll let you two fight it out 😂

Reviewer: hahahaha I won't fight, though I prefer not to duplicate the `Migrations` bit :-)

Reviewer: Ok, updating
